src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.Converters.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization.Converters;
+
+namespace System.Text.Json.Serialization.Metadata
+{
+    public static partial class JsonMetadataServices
+    {
+        /// <summary>
+        /// Returns a <see cref="JsonConverter{T}"/> instance that converts <see cref="bool"/> values.
+        /// </summary>
+        public static JsonConverter<bool> BooleanConverter => s_booleanConverter ??= new BooleanConverter();
```

Comment:
Similarly, tests would be nice to have for these methods. You could probably use a generic theory to avoid replicating tests for every type.

Reply:
Added tests for the `GetEnumConverter` and `GetNullableConverter` methods. The converter properties are used by the serializer's warm-up code-paths, so their functionality is already fully covered.